### ISSUE TITLE
For training details

### ISSUE THREAD

**Issue by xunge** [AustinDoolittle/Pytorch-Gain#2]
Hi:)
Really appreciate for your work~
I'm trying to train the GAIN model on ImageNet dataset based on resnet18 for 40 epoches, but the heatmap generated is not satisfied, so I have some questions.
1. I find that both your code and paper point that the batch size for training is 1, is it necessary?
2. The pretrained epoch default is 100, but it cost too much for me. Could I use the pretrained models?
3. Can you show me the heatmap generated yourself?

Thank you for a lot~

**Comment by AustinDoolittle** [comment]
Hi @xunge! This is a project that I haven't actively worked on for a couple years now, so unfortunately I don't have the pretrained model or any example outputs of this implementation. It is also important to mention that I was not able to recreate the results of the paper, but that could be because my implementation was incorrect. I did have some luck contacting the original author of the paper when I was implementing this, that may be a good route forward if you are interested in further pursuing this algorithm.

Thanks!

**Comment by AustinDoolittle** [comment]
And per batch size, I do not specifically remember the reason that the batch size was restricted to 1, but I think it had something to do with the online gradcam calculation... not totally sure

**Comment by AustinDoolittle** [comment]
You may also find some luck in discussing the project with some of the smart folks who have forked this project and extended the work that I have started!

**Comment by xunge** [comment]
Thanks for your reply, I think your work is correct and I need to try some other models. Appreciate for your work again.

**Comment by paganpasta** [comment]
@xunge Would you have the modified code for ImageNet dataset? I am trying to work with cifar10 using resnet18 and and having difficulty producing good top-1 accuracy.
